packages/shared/src/lib/core/network/utils/getExplorerApiNetworkName.ts

```diff
@@ -10,6 +10,6 @@ export function getExplorerApiNetworkName(networkId: NetworkId): string {
         case SupportedStardustNetworkId.Testnet:
             return 'testnet'
         default:
-            return ''
+            return 'iota-testnet'
```

Comment:
dont put this in the default, but rather in a 
` case SupportedStardustNetworkId.IotaTestnet`